### DISCUSSION TITLE
What kind of document is it that confirms academic status and why are there billing errors?

### DISCUSSION THREAD

**Discussion by BlackRandomCraft** [community/community#106559] [GitHub Education]
### Hi everyone,

[the document together with an English translation](https://github.com/community/community/assets/67829038/7174b524-4cd1-4c50-9efa-535f26f16b90)

Every time I submit a document, I always get the name wrong in the billing, even though it's all correct.
When I attach a document with a translation the error "Please upload proof of your current school affiliation" comes out, when without translation it starts uploading and the billing error comes out.

**Comment by ghost** [comment]
Hi. It's possible that there's a mismatch between the information provided in your document and the details required by the GitHub Education platform. Such mismatches could be related to how your name is spelled, the format of the dates!!!!!, or any number of fields that do not align precisely with their system's expectations. Double-check the information on the document to ensure it matches your personal details on GitHub Education exactly, including the spelling of names and the format of dates. Ensure that the document clearly states your current academic status and includes all the necessary details such as the institution's name, your full name, and the academic year.

**Reply by BlackRandomCraft** [reply]
I have an external passport with my name written in Russian and English, it's fine. I'll look at the dates, but I got my school certificate yesterday.
upd: but my patronymic is also written in the certificate, and I wrote it in the translation, and maybe because of it it doesn't pass.

**Comment by samus-aran** [comment]
Hi @BlackRandomCraft 👋 

Can you please provide a full list of your rejection reasons? 

Here is a few quick tips on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile/billing information. I would suggest this also if you have difficult to read handwriting.. try re-writing important info in **uppercase** and position it next to your evidence. If you are unable to read its text on the image itself then the verification process may not be able to either.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.
- We request students submit a live photo of their evidence rather than an upload (in special circumstances).

**Reply by BlackRandomCraft** [reply]
1. about the translation, I looked and it is readable, you can see for yourself in the header there is a photo of what I photographed.

2. about the name, there too everything is normal, maybe it does not pass because of the patronymic in the translation.

upd:<details>
<summary>errors</summary>

![image](https://github.com/community/community/assets/67829038/74096218-a607-44f3-823e-8401506ef48a)

![image](https://github.com/community/community/assets/67829038/a2b5a821-76c9-48c1-b0a8-3326320dce69)
some of the errors are repetitive, so I haven't attached them all.
</details>
